src/opera/test/pge/disp_s1/test_disp_s1_pge.py

```diff
@@ -28,6 +31,30 @@
 from opera.util.input_validation import validate_disp_inputs
 
 
+def mock_grib_to_netcdf(*popenargs, input=None, capture_output=False, timeout=None, check=False, **kwargs):
+    """
+    Mack grit_to_netcdf function.
```

Comment:
Should read "Mock grib_to_netcdf function", but what this function really is is a mock implementation for `subprocess.run`, and that should be noted in the documentation here.